tests/aws/services/events/test_events.py

```diff
@@ -708,6 +708,10 @@ def _get_sqs_messages():
 
 class TestEventBus:
     @markers.aws.validated
+    @pytest.mark.skipif(
+        not is_v2_provider() and not is_aws_cloud(),
```

Comment:
nit: Can we introduce a concise helper `is_old_provider()` (💡 old is a bit easier to distinguish than v1 vs v2) for readability?
(ok in follow-up)

Reply:
added to refactor test backlog as a task